{{ cookiecutter.__project_name_kebab }}/requirements-ci.txt

```diff
@@ -0,0 +1,2 @@
+tox==4.11.3
+tox-gh-actions==3.1.3
```

Comment:
Personally I am not a huge fan of just running `python -m pip install tox tox-gh-actions` with no pins. Obviously it is all fun and games.. until the day `tox==5.0.0` is released. I accept this is somewhat a matter of taste, but personally I prefer predictable builds. I also appreciate this adds another thing that everyone has to remember to bump (both as maintainers of the template and plugin authors who generate from it), and we all love bumping version numbers.

Reply:
IThe tox version problem would be the same if one would want to run the tests with tox

I wonder if it makes sense to move these to pyproject.toml in a CI optional dependency?
The upside is we limit the number of places to change. so then we can install everything with `[ci]`.. The downside, perhaps we do more than we need to.

Reply:
Yeah my initial thought was to put them under `testing` in `pyproject.toml`, but we delegate installing that to tox https://github.com/wagtail/cookiecutter-wagtail-package/blob/3a9d3d2f6190abb804185630c2edc2e07483453f/%7B%7B%20cookiecutter.__project_name_kebab%20%7D%7D/tox.ini#L10 but tox needs to be installed before it can run that for us :chicken: :egg: which led me to this.

Just to clarify: Is your suggestion that we add a `project.optional-dependencies.ci` or `project.optional-dependencies.tox` section to pyproject.toml, and then something like `pip install .[ci]` in the GH workflow before we run tox (the downside being we install the package itself twice)? If so, I think you're right that is preferable. It keeps everything in one file.

Reply:
Agreed that is the best solution. I'll update to use that approach

Reply:
done in e3b9175